whylog/constraints/__init__.py

```diff
@@ -191,6 +217,15 @@ def _verify_both(self, group_contents, param_dict):
     def verify(self, group_contents, param_dict):
         pass
 
+    def _validate_params(self):
+        problems = super(TimeConstraint, self)._validate_params()
+        param_min_delta = self.params.get(self.MIN_DELTA)
+        param_max_delta = self.params.get(self.MAX_DELTA)
+        if param_min_delta is not None and param_max_delta is not None:
+            if param_min_delta > param_max_delta:
+                problems.append(MinGreaterThatMaxProblem())
+        return problems
```

Comment:
if no deltas are specified, we report no problem here and trigger `WrongConstraintClassSetup` in runtime